NEWS

```diff
@@ -11,6 +11,7 @@ Bug fixes
  - Fix exception modifications import in recurrence series
  - Sent mails are not longer in Drafts folder using Outlook
  - Fix server side crash parsing rtf emails with images (with word97 format)
+ - Internet headers are now shown in Outlook
```

Comment:
As I've already told you in another pull request, this is an Invalid position to put the change, either move to top or put it on pull request (and I will add it on merge commit)